### PR TITLE
Release notes for v2.1.1

### PR DESCRIPTION
**Pull request by pavansshanbhag**
Please review
Signed-off-by: Pavan Shanbhag <pavan-s-shanbhag@hpe.com>

### CODE REVIEW

File: release-notes/v2.1.1.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.1.1.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|CON-2445|csi.k8s|Multipathing: "device not found with serial < wwid > or target" error after node reboot where csp pod is running|For 3PAR devices, the new volume(device) is not discovered due to a LUN remapping scenario playing out. A fix has been made to handle LUN remapping scenario, by modifying the Device Serial Number matching algorithm for 3PAR devices, to discover stale multipathing entries, and deleting them|
+|CON-2446|csi.k8s|All the PVC are getting disconnected when one of POD is deleted on a worker node.|This bug has been root caused to have been caused by the LUN remapping issue as in CON-2445|
+|CON-2590|csi.k8s|Security vulnerabilities found in HPE CSI Driver images| The base images were upgraded from ubi-7.9 to ubi 8.5, which fixes majority of the security vulnerabilities identified in CSI driver 2.1.0|
+|CON-2595|csi.k8s| Reasonable defaults to be set for optimal performance| Performance benchmarked configurations have been set by default, for optimal performance|
```
Comment (raunakkumar) [inline release-notes/v2.1.1.md:18]: can we mention what performance parameter defaults were set?

File: release-notes/v2.1.1.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# HPE CSI Driver for Kubernetes 2.1.1 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.1.1
+
+| **Version:** |2.1.1|
+|--------------|-----|
+| **Revision:** | Tuesday February 24, 2022 |
```
Comment (datamattsson) [inline release-notes/v2.1.1.md:7]: Thursday.

**Review by datamattsson — COMMENTED** [review]
LGTM

**Comment by datamattsson** [comment]
@pavansshanbhag @raunakkumar ?

**Comment by raunakkumar** [comment]
> @pavansshanbhag @raunakkumar ?

Was waiting on update from @pavansshanbhag .

**Review by raunakkumar — APPROVED** [review]

**Pull request merged**